### PR TITLE
NMS-8552: Added documentation for Eventd's configuration files.

### PR DESCRIPTION
**Pull request by StCyr**
- JIRA: http://issues.opennms.org/browse/NMS-8552
- Bamboo: Do not worry, we add a link to our continuous integration system here

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/index.adoc [hunk 1]
```diff
@@ -172,6 +172,12 @@ include::text/operation/newts/configuration.adoc[]
 include::text/operation/newts/cassandra-jmx.adoc[]
 include::text/operation/newts/cassandra-newts-jmx.adoc[]
 
+[[ga-opennms-operation-daemon-config-files]]
+=== Daemon Configuration Files
+include::text/operation/daemon-config-files/introduction.adoc[]
```
Comment (RangerRick) [inline opennms-doc/guide-admin/src/asciidoc/index.adoc]: Where are introduction.adoc and pollerd.adoc?  They're not in this PR, jus the eventd.adoc is.

Comment (StCyr) [inline opennms-doc/guide-admin/src/asciidoc/index.adoc]: Hi @RangerRick 

They are here https://github.com/OpenNMS/opennms/pull/890 and here https://github.com/OpenNMS/opennms/pull/891

I don't know how to handle this, sorry.

**Comment by RangerRick** [comment]
Ah, I see.  So those need to be merged first, then this one? Nevermind then.  :)

**Comment by indigo423** [comment]
Sorry, this pull request depends on #890 and needs to be merged first.

**Comment by StCyr** [comment]
Hi @indigo423 ,

Is the process OK though? 

May I proceed the same way with provisiond?

**Comment by mvrueden** [comment]
Hey Ulf, please check the OCA.

**Comment by j-white** [comment]
PR #890 was merged, we just need to fix the conflicts here now.

**Comment by j-white** [comment]
@indigo423 can you take care fixing the merge conflicts here? :smile:

**Comment by indigo423** [comment]
Resolved merge conflicts and reopened in #1136.

**Pull request closed without merging**